rule-packs/gcp.json

```diff
@@ -28,7 +28,7 @@
         "queries": [
           {
             "name": "query0",
-            "query": "find google_iam_service_account as user\n  that ASSIGNED google_iam_role\n  with name!~=(\n    \"admin\" and\n    \"Admin\"\n  ) and name!=(\n    \"roles/owner\" and\n    \"roles/editor\"\n  )\nreturn\n  user.name,\n  user.username,\n  user.email,\n  google_iam_role.name",
```

Comment:
These questions were written incorrectly originally, so I fixed them as well. What they were previously searching for was the OPPOSITE of the query description

> Finds service accounts with admin privileges.

instead, it was finding all service accounts WITHOUT admin privileges.

Query originally taken incorrectly from here - https://github.com/JupiterOne/graph-google-cloud/blob/main/jupiterone/questions/questions.yaml#L67